src/bionics.cpp

```diff
@@ -280,6 +280,13 @@ void bionic_data::load( const JsonObject &jsobj, const std::string )
 
     optional( jsobj, was_loaded, "available_upgrades", available_upgrades );
 
+    if( jsobj.has_string( "installation_requirement" ) ) {
+        std::string temp_string;
+        optional( jsobj, was_loaded, "installation_requirement", temp_string );
+        instalation_requirement = requirement_id( temp_string );
+    }
+
+
```

Comment:
```suggestion
    optional( jsobj, was_loaded, "installation_requirement", instalation_requirement );
```
that misspelling of installation_requirement in the variable name is going to bug me. also, this should work for string_ids just fine.